docs/guides/creating-a-c++-mod.md

```diff
@@ -17,27 +17,44 @@ Part four will cover installation of the mod.
 5. Open CMD and cd into `RE-UE4SS` and execute: `git submodule update --init --recursive`
 6. Go back to the `MyMods` directory and create a new directory, this directory will contain your mod source files.
 I named mine `MyAwesomeMod`.
-7. Create a file called `CMakeLists.txt` inside `MyMods` and put this inside it:
-```cmake
-cmake_minimum_required(VERSION 3.18)
-
-project(MyMods)
-
-add_subdirectory(RE-UE4SS)
-add_subdirectory(MyAwesomeMod)
+7. Create a file called `xmake.lua` inside `MyMods` and put this inside it:
+```lua
+includes("RE-UE4SS")
+includes("MyAwesomeMod")
 ```
 
 ## Part #2
-1. Create a file called `CMakeLists.txt` inside `MyMods/MyAwesomeMod` and put this inside it:
-```cmake
-cmake_minimum_required(VERSION 3.18)
-
-set(TARGET MyAwesomeMod)
-project(${TARGET})
-
-add_library(${TARGET} SHARED "dllmain.cpp")
-target_include_directories(${TARGET} PRIVATE .)
-target_link_libraries(${TARGET} PUBLIC UE4SS)
+1. Create a file called `xmake.lua` inside `MyMods/MyAwesomeMod` and put this inside it:
+```lua
+local projectName = "MyAwesomeMod"
+
+target(projectName)
+    set_kind("shared")
+    set_languages("cxx20")
+    set_exceptions("cxx")
+
+    add_includedirs(".")
+
+    add_files("dllmain.cpp")
+
+    add_deps("UE4SS")
+
+    after_load(function (target)
+        import("build_configs", { rootdir = get_config("scriptsRoot") })
+        import("target_helpers", { rootdir = get_config("scriptsRoot") })
+        build_configs:set_output_dir(target)
+        build_configs:export_deps(target)
```

Comment:
`build_configsLexport_deps(target)` already runs as part of the UE4SS target, so I believe this is the root of the current linker issue where there are multiply defined symbols.